pravega/pravega
# Issue 502: Basic flink source

**Change log description**
Implementation of a simple flink source connector for reading from pravega.
* No support for checkpointing, hence no support for flink job failures.

**Purpose of the change**
To provide a flink connector for reading events from pravega streams

**What the code does**
Uses the pravega reader group apis to read from pravega streams in parallel flink jobs

**How to verify it**
Run a flink job with the source as this connector. 
See integration tests for examples. Tests with parallelism > 1 fail inconsistently in the reader implementation, hence added tests only with parallelism 1 for now.
